.vscode/tasks.json

```
@@ -226,12 +227,14 @@
         },
         {
             "label": "Setup Secrets",
```

```suggestion
            "label": "Set Up Secrets",
```

⛏️ but can't resist. Doesn't need to actually be applied but noun vs. verb mistakes always get me.